docs/docs/tutorials/creating-cds-dashboard.md

```diff
@@ -0,0 +1,94 @@
+---
+id: creating-cds-dashboard
+title: How to create a ContainDS dashboard on Nebari
+---
+
+# What is a dashboard?
+
+Analyzing data provides us with insights. A dashboard stitches these insights together into a meaningful story.
+
+Different tools and technologies can be used to create and deploy a dashboard. In this tutorial
+we will be building a simple dashboard on _Nebari_ which can be shared with other users and groups via
+[ContainDS Dashboards](https://cdsdashboards.readthedocs.io/en/stable/) (CDS Dashboards).
+
+Currently, CDS Dashboards supports `Panel`, `Bokeh`, `Voila`, `Streamlit`, and `Plotly`.
+We will be using `Panel` in this tutorial to create a dashboard.
+
+# Let's get our hands dirty
+
+- Create a notebook in your JupyterLab environment on Nebari with a name of your choice
+- Select an environment from the select kernel dropdown (located on the top right: e.g. `dashboard`)
+- Copy the code below to your notebook.
+
+We can now execute the notebook and view our Panel dashboard as output right inside the notebook.
+Making it super easy to rapidly prototype new dashboards. Once we are happy with what we've created,
+we can move on to deploying it with CDS Dashboards.
+
+The code below dives into creating a dashboard from insights.
+
+##### Creating insights using `holoviews`, `panel` and `bokeh` as backend
+
+```python
+# necessary imports
+import numpy as np
+import pandas as pd
+import holoviews as hv
+from bokeh.models import HoverTool
+from holoviews import opts
+import panel as pn
+hv.extension('bokeh')
+
```

Comment:
are you sure you don't need a `pn.extension()` here? I think leaving it off means this dashboard will work in some cases, but not in others.

Reply:
For the last few EDA/dashboards, I didn't include `pn.extension()` and didn't face any issues. 
Though I have updated the code for the tutorial (to eliminate any unforeseen errors)